jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -1350,6 +1350,9 @@ ecma_op_function_construct (ecma_object_t *func_obj_p, /**< Function object */
     /* 5.b */
     if (JERRY_UNLIKELY (proto_p == NULL))
     {
+#if ENABLED (JERRY_ESNEXT)
```

Comment:
I was wondering: if we move the `1340-1341` lines from above just before the `.._call_simple` to avoid adding this here ? Would that change anything in the `ecma_op_get_prototype_from_constructor` or `ecma_create_object` internals?

Reply:
That's a good idea.